test/utils/InstanceTest.sol

```diff
@@ -927,8 +1001,15 @@ abstract contract InstanceTest is HyperdriveTest {
         }
 
         // Ensure Bob is credited the correct amount of bonds.
-        assertLe(baseProceeds, longAmount);
-        assertApproxEqAbs(baseProceeds, longAmount, 10);
+        assertLe(
+            baseProceeds,
+            longAmount.mulDown(ONE - hyperdrive.getPoolConfig().fees.flat)
+        );
+        assertApproxEqAbs(
+            baseProceeds,
+            longAmount.mulDown(ONE - hyperdrive.getPoolConfig().fees.flat),
```

Comment:
was this able to pass earlier due to the flat fee being smaller than the "approx" tolerance?

good catch!!!!

Reply:
Yeah, it was